index.html

```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
 			</dd>
-			<dt><dfn data-export="">Taxonomy</dfn></dt><!--Not used-->
+			<dt><dfn>Taxonomy</dfn></dt><!--Not used-->
 			<dd>
-			  <p>A hierarchical definition of how the characteristics of various <a class="termref" data-lt="class">classes</a> relate to each other, in which classes inherit the properties of superclasses in the hierarchy. A taxonomy can comprise part of the formal definition of an <a>ontology</a>.</p>
+			  <p>A hierarchical definition of how the characteristics of various classes relate to each other, in which classes inherit the properties of superclasses in the hierarchy. A taxonomy can comprise part of the formal definition of an <a>ontology</a>.</p>
 			</dd>
-			<dt><dfn data-export="">Text node</dfn></dt>
+			<dt><dfn>Text node</dfn></dt>
 			<dd>
 			  <p>Type of <abbr title="Document Object Model">DOM</abbr> <a class="termref" data-lt="node">node</a> that represents the textual content of an <a class="termref">attribute</a> or an <a class="termref">element</a>. A Text node has no child nodes.  </p>
 			</dd>
-			<dt><dfn data-export="">Tooltip attribute</dfn></dt><!--Accname-->
+			<dt><dfn>Tooltip attribute</dfn></dt><!--Accname-->
 			<dd>
 			  <p>Any host language attribute that would result in a user agent generating a tooltip such as in response to a mouse hover in desktop user agents.</p>
 			</dd>
-			<dt><dfn data-export="">Understandable</dfn></dt><!--Not used-->
+			<dt><dfn>Understandable</dfn></dt><!--Not used-->
 			<dd>
 			  <p>Presentable to users in ways they can construct an appropriate meaning. References in this document relate to <cite><a href="https://www.w3.org/TR/WCAG21/#understandable"><abbr title="Web Content Accessibility Guidelines">WCAG</abbr> 2.1 Principle 3: Information and the operation of user interface must be understandable</a></cite> [[WCAG21]].</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="unicode braille">Unicode Braille Patterns</dfn></dt>
+			<dt><dfn data-lt="unicode braille">Unicode Braille Patterns</dfn></dt>
 			<dd>
 				<p>In Unicode, braille is represented in a block called Braille Patterns (U+2800..U+28FF). The block contains all 256 possible patterns of an 8-dot braille cell; this includes the complete 6-dot cell range which is represented by U+2800..U+283F. In all braille systems, the braille pattern dots-0 (U+2800) is used to represent a space or the lack of content; it is also called a blank Braille pattern. See <cite><a href="https://en.wikipedia.org/wiki/Braille_Patterns">Braille Patterns on Wikipedia</a>.</cite></p>
 			</dd>
-			<dt><dfn data-export="" data-lt="user agent|user agents">User Agent</dfn></dt>
+			<dt><dfn data-lt="user agent|user agents">User Agent</dfn></dt>
 			<dd>
 			  <p>Any software that retrieves, renders and facilitates end user interaction with Web content. This definition may differ from that used in other documents.</p>
 			</dd>
-			<dt><dfn data-export="">Valid IDREF</dfn></dt>
+			<dt><dfn>Valid IDREF</dfn></dt>
```

Comment:
not sure we need to define this at all. IMO we should remove the one reference to it and delete the definition